search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/client/ConversationalMemoryClient.java

```diff
@@ -37,6 +42,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.function.BiFunction;
```

Comment:
Did we use these newly imported libraries?

Reply:
Removed.